FordLabs/PeopleMover
# 289 space name not unique

## Issue
Connects #289

## What was done
- [x] Change url to use uuid for space instead of name 

## How to test
1. Go to space and verify you see a bunch of characters instead of name in url
2. Make sure all app features still work
